truss/tests/test_truss_util.py

```diff
@@ -16,4 +16,49 @@ def test_max_modified():
         assert t2 > t1
 
 
+def test_load_gitignore_patterns():
```

Comment:
Can you please add a test that:
1. takes `custom_model_truss_dir_with_hidden_files`
2. runs a predict on it (or any operation that requires `gather`)
3. assert that the original truss is not modified
4. assert that the ignored files are not in the gathered truss